front/src/modules/timesStops/TimeInput.tsx

```diff
@@ -26,7 +27,7 @@ const TimeInput = ({
     setTempTimeValue(rowData);
   }, [rowData]);
 
-  return (
+  const input = (
     <input
       className="dsg-input"
```

Comment:
I think it was made so that `react-datasheet-grid` can use it, it’s classes used inside the library.

https://github.com/nick-keller/react-datasheet-grid/blob/master/src/style.css